### PR TITLE
Update 3ds.md

### PR DESCRIPTION
**Pull request by Yevnilc-S-C**
Added a warning to tell users not to make the NNID and PNID the same, since the wii u install doc had the warning but the 3ds was missing it, causing some users to accidentally create an account with the same details as their NNID

### CODE REVIEW

**Comment by jonbarrow** [comment]
You're working on the wrong branch

File: docs/en_US/install/3ds.md [hunk 1]
```diff
@@ -91,6 +91,8 @@ Setting up a PNID on the 3DS is the same as setting up a NNID. You may either cr
 
 It is recommended to register the PNID on your device at this time, as registering on the website does not currently allow you to change your user data
 
+CAUTION: A Pretendo Network ID may not be the same as the one currently linked to your 3ds, ensure that your PNID and NNID do not match before you make it.
```
Comment (jonbarrow) [inline docs/en_US/install/3ds.md:94]: Please make this into a call out, you can just copy-paste it from the Wii U docs

**Review by jonbarrow — CHANGES_REQUESTED** [review]

File: docs/en_US/install/3ds.md (hunk 1 above)
Comment (Yevnilc-S-C) [reply docs/en_US/install/3ds.md:94]: Oh shoot, I'm dumb, haha, I assume I instead work on the dev branch, I'll go fix that rq

**Comment by jonbarrow** [comment]
Closing as #275 was merged already

**Pull request closed without merging**